io.github.chris2511.xca.appdata.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>io.github.chris2511.xca</id>
+    <name>XCA</name>
+    <summary>X Certificate and Key management</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LicenseRef-proprietary=https://github.com/chris2511/xca/raw/master/COPYRIGHT</project_license>
+    <provides>
+        <binary>xca</binary>
+    </provides>
+    <launchable type="desktop-id">io.github.chris2511.xca.desktop</launchable>
+    <categories>
```

Comment:
Categories are taken from the desktop file, no need to specify them here.

Reply:
Removed <categories> as requested.